### DISCUSSION TITLE
Admin password

### DISCUSSION THREAD

**Discussion by evgkg** [DependencyTrack/dependency-track#3032] [Q&A]
can I disable changing the administrator password at the first startup?
If there is no such possibility, then how can I change the admin password through the api? The handle "ForceChangePassword" is not suitable, because it returns "Unauthorized"

**Comment by syalioune** [comment]
Hello @evgkg 

> Can I disable changing the admin password at first startup ?

That's not possible at the moment. It will when https://github.com/DependencyTrack/dependency-track/issues/2443 is merged (target is next release)

> How can I change the admin password through the api

See https://github.com/DependencyTrack/dependency-track/issues/831#issuecomment-725971882
This should work.
```shell
$ curl \
    -d "username=admin" \
    -d "password=admin" \
    -d "newPassword=admin123" \
    -d "confirmPassword=admin123" \ 
    "$DTRACK_URL/api/v1/user/forceChangePassword"
```

**Reply by evgkg** [reply]
Thanks!
